Resources/ConfigPresets/DeltaV/apoapsis.toml

```
@@ -1,6 +1,7 @@
 [game]
 hostname         = "[EN][MRP] Delta-v (Ψ) | Apoapsis [US East 1]"
 soft_max_players = 80
+round_restart_time = 30
 
```

```suggestion
round_restart_time = 30
lobbyduration    = 360
```
Please, consider giving some extra time between rounds. Before you had 9 minutes skipping EOR + even time on the shuttle if you needed it. Now it's just 4 minutes, or 4 minutes 15 seconds worst case from evac leaving to next round starting.